server/kv.go

```diff
@@ -644,6 +644,11 @@ func (store store) OneTimeLoadOauth1aTemporaryCredentials(mmUserId string) (*OAu
 	if appErr != nil {
 		return nil, errors.WithMessage(appErr, "failed to load temporary credentials for "+mmUserId)
 	}
+	// If the key expired, appErr is nil, but the data is also nil
+	if len(b) == 0 {
```

Comment:
Nit pick: The check should be if the data was not found, not if it's an empty slice.
```suggestion
	if b == nil {
```